### PR TITLE
Adding CVE-2018-6491

### PR DESCRIPTION
**Pull request by cyberpsrt**

### CODE REVIEW

File: 2018/6xxx/CVE-2018-6491.json [hunk 1]
```diff
@@ -1,18 +1,96 @@
 {
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6491.json]: JSON: u'url' is a required property

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.

**Comment by cyberpsrt** [comment]
I believe I fixed the error. Can you confirm?





Best Regards,



Diana Della Rocchetta

Global Assessment and Assurance Services

PSRT

File: 2018/6xxx/CVE-2018-6491.json [hunk 2]
```diff
@@ -1,18 +1,93 @@
...
+      }
+   ],
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
+         {
+            "lang": "eng",
+            "value": "Local Escalation of Priviledge vulnerability to Micro Focus Univeral CMDB, versions 10.20, 10.21, 10.22, 10.30, 10.31, 10.32, 10.33, 11.00. The vulnerability could be remotely exploited to Local Escalation of Privilege."
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6491.json]: There's a typo in the description ("Univeral"). I'll fix that when I populate the entry.

George

**Pull request merged**